pkg/infrastructure/azure/storage.go

```diff
@@ -73,8 +74,8 @@ func CreateStorageAccount(ctx context.Context, in *CreateStorageAccountInput) (*
 
 	opts := &arm.ClientOptions{
 		ClientOptions: policy.ClientOptions{
-			Cloud:      in.ClientOpts.Cloud,
-			APIVersion: "2019-06-01",
+			Cloud: in.ClientOpts.Cloud,
+			//APIVersion: "2019-06-01", //apparently override is not supported
```

Comment:
clean this up

Reply:
The comment might be still helpful as a warning :D I guess you decided to leave it.